src/stamps.ts

```diff
@@ -5,26 +5,46 @@ import { sleep } from './utils'
 const logger = console // eslint-disable-line no-console
 
 const DEFAULT_POLLING_FREQUENCY = 1_000
+const DEFAULT_STAMP_USABLE_TIMEOUT = 10_000
```

Comment:
Do you have some data on how fast stamps get usable? I think it should be more then 10 seconds no? I have it somewhere in my mind that it can take up to 30 seconds no?

Reply:
No I don't. I agree this is probably more, maybe even 60 seconds.